stablehlo/dialect/StablehloOps.td

```diff
@@ -192,16 +192,7 @@ def StableHLO_AbsOp: StableHLO_UnaryElementwiseOp<"abs",
      TensorOf<[HLO_SInt, HLO_Float, HLO_Complex]>> {
   let summary = "Absolute value operator";
   let description = [{
-    Returns `abs(operand)` element-wise.
-
-    See
-    https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions.
-
-    Example:
-
-    ```mlir
-    %0 = stablehlo.abs %arg0 : tensor<3xi32>
-    ```
```

Comment:
@GleasonK What's your take on how our documentation in the ODS should look like? I think it's a reasonable idea to reduce it to just a link to the spec if it exists, but this leaves an open question on what we should do with pretty syntax examples (because currently that's not part of the spec). What do you think?

Reply:
I see no reason to avoid pretty syntax in the doc examples, that could suffice. For example, the current doc has the following:

```
// integers
// %operand: [-2, 0, 2]
%result = "stablehlo.abs"(%operand) : (tensor<3xi32>) -> tensor<3xi32>
// %result: [2, 0, 2]
```

Could be:

```
// integers
// %operand: [-2, 0, 2]
%result = stablehlo.abs %operand : tensor<3xi32>
// %result: [2, 0, 2]
```

It is worth noting that removing this from `.td` files causes some incongruity between MHLO/StableHLO, since MHLO doc is generated directly from the ODS, so the pretty print example should appear in the `.td` for MHLO. However, given that this is not a design goal to keep StableHLO/MHLO docs in sync, this is fine with me.

Reply:
The reason for using the extended form is noted in #191, but indeed our ODS and spec doc do not provide consistency for which form is recommended.

Reply:
I propose that we don't include prettyprinting in the spec because that'll be one more thing that spec readers will need to know about.

However, on a second thought, I think it would be good to keep pretty syntax in the ODS because that's part of the op's documentation. How about instead of removing everything from the description, we keep: 1) one sentence that describes the op, 2) pretty syntax, 3) link to spec?

Reply:
SGTM